.gitattributes

```diff
@@ -0,0 +1,11 @@
+/.atoum.php           export-ignore
+/.bootstrap.atoum.php export-ignore
+/.coke                export-ignore
+/.gitattributes       export-ignore
+/.gitignore           export-ignore
+/.travis.yml          export-ignore
+/bin                  export-ignore
+/Dockerfile           export-ignore
+/docker               export-ignore
+/docker-compose.yml   export-ignore
+/travis               export-ignore
```

Comment:
By curiosity, in which case is it interesting to prevent the export of these files? 🤔 
Is it to provide lighter composer package? If so, what about `src/AmqpBundle/Tests/`?
Just wondering 😄

Reply:
> Is it to provide lighter composer package?

indeed :D 

https://github.com/M6Web/AmqpBundle/pull/52#issuecomment-582836665